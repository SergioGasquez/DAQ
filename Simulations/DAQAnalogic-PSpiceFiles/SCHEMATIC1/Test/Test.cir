** Profile: "SCHEMATIC1-Test"  [ c:\users\sergi\documents\github\daq\simulations\daqanalogic-pspicefiles\schematic1\test.sim ] 

** Creating circuit file "Test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
* From [PSPICE NETLIST] section of C:\Users\sergi\AppData\Roaming\SPB_Data\cdssetup\OrCAD_PSpice\17.2.0\PSpice.ini file:
.lib "nomd.lib" 

*Analysis directives: 
.DC LIN V_V1 0 5 0.01 
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 


.END
