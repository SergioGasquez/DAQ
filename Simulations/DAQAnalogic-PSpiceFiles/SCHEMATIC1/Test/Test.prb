[DISPLAYS]
BEGIN DISPLAY LAST SESSION
ANALYSIS DC_SWEEP
SYMBOL ALWAYS
TRACECOLORSCHEME NORMAL
BEGIN CURSORS
CURSORNUMDIGITS 5
BEGIN CURSOR 1
CURSOR_X 4.98999999999994
CURSOR_Y 12.0951578560476
ANAPLOT 1
ANATRACE 0
END CURSOR 1
BEGIN CURSOR 2
CURSOR_X 0
CURSOR_Y -13.2640774960038
ANAPLOT 1
ANATRACE 0
END CURSOR 2
END CURSORS
BEGIN ANAPLOT 1
ACTIVE
XBASE
BEGIN XAXIS
XAXISUSERNAME 0 (null)
RANGEFLAG AUTO
TYPE LINEAR
UNIT V
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE LINES
MINORPATTERN DOT
END GRIDS
END XAXIS
BEGIN YAXIS 1
YAXISSIDE LEFT
ACTIVE
RANGEFLAG AUTO
TYPE LINEAR
UNIT V
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE LINES
MINORPATTERN DOT
END GRIDS
BEGIN TRACE V(Vo)
MARKERID 27
END TRACE V(Vo)
BEGIN TRACE V(V1:+)
MARKERID 28
END TRACE V(V1:+)
END YAXIS 1
END ANAPLOT 1
END DISPLAY LAST SESSION
