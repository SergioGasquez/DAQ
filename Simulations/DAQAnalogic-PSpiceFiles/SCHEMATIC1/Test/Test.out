
**** 04/05/18 19:35:48 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-Test"  [ C:\Users\sergi\Documents\GitHub\DAQ\Simulations\daqanalogic-pspicefiles\schematic1\test.sim ] 


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "Test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
* From [PSPICE NETLIST] section of C:\Users\sergi\AppData\Roaming\SPB_Data\cdssetup\OrCAD_PSpice\17.2.0\PSpice.ini file:
.lib "nomd.lib" 

*Analysis directives: 
.DC LIN V_V1 0 5 0.01 
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 



**** INCLUDING SCHEMATIC1.net ****
* source DAQANALOGIC
V_V4         N01178 GND 15Vdc
V_V1         N03077 GND 5Vdc
V_V3         N01344 GND -15Vdc
R_R1         N03038 N01204  100k TC=0,0 
X_U2B         N01204 N01300 N01178 N01344 N01286 LM358
R_R2         N01204 GND  50k TC=0,0 
R_R5         GND N01300  47k TC=0,0 
R_R3         N013200 N01204  200k TC=0,0 
R_R4         N01300 N01286  100k TC=0,0 
V_V2         N013200 GND -5Vdc
R_R7         N02575 VO  50k TC=0,0 
V_V8         N02583 GND -15Vdc
V_V7         N02515 GND 15Vdc
X_U3B         N01286 N02575 N02515 N02583 VO LM358
R_R6         GND N02575  10k TC=0,0 
X_U4A         N03077 N03038 N03125 N03218 N03038 LM358
V_V9         N03125 GND 15Vdc
V_V10         N03218 GND -15Vdc

**** RESUMING Test.cir ****
.END

**** 04/05/18 19:35:48 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-Test"  [ C:\Users\sergi\Documents\GitHub\DAQ\Simulations\daqanalogic-pspicefiles\schematic1\test.sim ] 


 ****     Diode MODEL PARAMETERS


******************************************************************************




               X_U2B.x_lm358.dx 
          IS  800.000000E-18 
          RS    1            


               X_U2B.x_lm358.dy 
          IS  800.000000E-18 
          RS    1.000000E-03 
         CJO   10.000000E-12 


               X_U3B.x_lm358.dx 
          IS  800.000000E-18 
          RS    1            


               X_U3B.x_lm358.dy 
          IS  800.000000E-18 
          RS    1.000000E-03 
         CJO   10.000000E-12 


               X_U4A.x_lm358.dx 
          IS  800.000000E-18 
          RS    1            


               X_U4A.x_lm358.dy 
          IS  800.000000E-18 
          RS    1.000000E-03 
         CJO   10.000000E-12 


**** 04/05/18 19:35:48 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-Test"  [ C:\Users\sergi\Documents\GitHub\DAQ\Simulations\daqanalogic-pspicefiles\schematic1\test.sim ] 


 ****     BJT MODEL PARAMETERS


******************************************************************************




               X_U2B.x_lm358.qx 
               PNP             
       LEVEL    1            
          IS  800.000000E-18 
          BF  166.7          
          NF    1            
          BR    1            
          NR    1            
         ISS    0            
          RE    0            
          RC    0            
         CJE    0            
         VJE     .75         
         CJC    0            
         VJC     .75         
         MJC     .33         
        XCJC    1            
         CJS    0            
         VJS     .75         
          KF    0            
          AF    1            
          CN    2.2          
           D     .52         


               X_U3B.x_lm358.qx 
               PNP             
       LEVEL    1            
          IS  800.000000E-18 
          BF  166.7          
          NF    1            
          BR    1            
          NR    1            
         ISS    0            
          RE    0            
          RC    0            
         CJE    0            
         VJE     .75         
         CJC    0            
         VJC     .75         
         MJC     .33         
        XCJC    1            
         CJS    0            
         VJS     .75         
          KF    0            
          AF    1            
          CN    2.2          
           D     .52         


               X_U4A.x_lm358.qx 
               PNP             
       LEVEL    1            
          IS  800.000000E-18 
          BF  166.7          
          NF    1            
          BR    1            
          NR    1            
         ISS    0            
          RE    0            
          RC    0            
         CJE    0            
         VJE     .75         
         CJC    0            
         VJC     .75         
         MJC     .33         
        XCJC    1            
         CJS    0            
         VJS     .75         
          KF    0            
          AF    1            
          CN    2.2          
           D     .52         


**** 04/05/18 19:35:48 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-Test"  [ C:\Users\sergi\Documents\GitHub\DAQ\Simulations\daqanalogic-pspicefiles\schematic1\test.sim ] 


 ****     Current Controlled Switch MODEL PARAMETERS


******************************************************************************




               X_U2B.x_lm358.sw2 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 


               X_U2B.x_lm358.sw1 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U2B.x_lm358.sw3 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U2B.x_lm358.sw4 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 


               X_U3B.x_lm358.sw2 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 


               X_U3B.x_lm358.sw1 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U3B.x_lm358.sw3 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U3B.x_lm358.sw4 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 


               X_U4A.x_lm358.sw2 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 


               X_U4A.x_lm358.sw1 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U4A.x_lm358.sw3 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U4A.x_lm358.sw4 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 



          JOB CONCLUDED

**** 04/05/18 19:35:48 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-Test"  [ C:\Users\sergi\Documents\GitHub\DAQ\Simulations\daqanalogic-pspicefiles\schematic1\test.sim ] 


 ****     JOB STATISTICS SUMMARY


******************************************************************************



  Total job time (using Solver 1)   =         .08
